docs/source/_static/style_sidebar_brand.css

```diff
@@ -0,0 +1,11 @@
+.sidebar-sticky .sidebar-brand {
+      flex-direction: row;
+}
+
+.sidebar-sticky .sidebar-brand .sidebar-logo-container {
+      align-self: center;
+}
+
+.sidebar-sticky .sidebar-brand .sidebar-brand-text {
+      align-self: center;
```

Comment:
Maybe it makes a difference on mobile, but on PC I see no difference with or without these directives

Reply:
this only ensures that the logo and text are vertically centered. The change is very subtle since both are almost of the same size. You can see the difference if you open the dev tools of your browser and toggle the setting :)